apps/web/src/studio/components/workflows/test-workflow/WorkflowsTestPage.tsx

```diff
@@ -18,31 +18,28 @@ import { useTemplateFetcher } from '../../../../api/hooks/index';
 import { useSegment } from '../../../../components/providers/SegmentProvider';
 import { useStudioState } from '../../../StudioStateProvider';
 import { testTrigger } from '../../../../api/notification-templates';
-import { useApiKeys } from '../../../../hooks/useNovuAPI';
 
 export const WorkflowsTestPage = () => {
   const segment = useSegment();
-  const { isLocalStudio: local, testUser } = useStudioState() || {};
+  const studioState = useStudioState() || {};
```

Comment:
Adding `studioState` on function scope to later retrieve `tunnelBridgeUrl` via `isLocalStudio===true` type discriminator in local mode